core/services/keeper/validate.go

```diff
@@ -40,7 +40,8 @@ perform_upkeep_tx        [type=ethtx
                           evmChainID="$(jobSpec.evmChainID)"
                           data="$(encode_perform_upkeep_tx)"
                           gasLimit="$(jobSpec.performUpkeepGasLimit)"
-                          txMeta="{\"jobID\":$(jobSpec.jobID),\"upkeepID\":$(jobSpec.upkeepID)}"]
+                          txMeta="{\"jobID\":$(jobSpec.jobID),\"upkeepID\":$(jobSpec.upkeepID)}"
+                          simulate=true]
```

Comment:
curious, where is this parameter consumed? Couldn't find it in ethtx: https://github.com/smartcontractkit/chainlink/blob/develop/core/services/pipeline/task.eth_tx.go#L70-L76

Reply:
Yes, you have to pass a transmit checker now